snakemake_executor_plugin_googlebatch/command.py

```diff
@@ -31,38 +31,31 @@
 )
 
 install_snakemake = """
-# Only the main job should install conda (rest can use it)
 echo "I am batch index ${BATCH_TASK_INDEX}"
 export PATH=/opt/conda/bin:${PATH}
-if [ $BATCH_TASK_INDEX = 0 ] && [ ! -d "/opt/conda" ] ; then
```

Comment:
I'm not entirely sure about best practice here - if we should install snakemake just to the main executor (running the workflow) or as I've done, allow installing to all. For the other jobs (that ran OK) I was just installing to index 0. We can run a test again after this and see if the previous are still working OK. Intuitively unless there is an obvious shared filesystem (there isn't at /opt) I would want snakemake installed across nodes.